spec/integration/apply_spec.rb

```diff
@@ -218,9 +218,9 @@
           end
         end
 
-        # Run on puppet_6_node and puppet_7_node only, as deferred requires >= 6.
+        # Run on puppet_7_node only, as deferred requires >= 6.
```

Comment:
This makes me think this was in place to support Puppet 5.  Maybe we can drop this comment completely?

Reply:
Yeah I nearly just deleted the comment. Good call, I’ll remove it